bundlesize.config.json

```
@@ -6,7 +6,7 @@
     },
     {
       "path": "packages/autocomplete-js/dist/umd/index.production.js",
-      "maxSize": "15.25 kB"
+      "maxSize": "15.3 kB"
```

We usually increase with `0.25` steps.

```suggestion
      "maxSize": "15.5 kB"
```